### ISSUE TITLE
Split up documentation

### ISSUE THREAD

**Issue by allmarkedup** [allmarkedup/lookbook#85]
The single-page readme approach is getting a bit hard to parse now. Until a proper docs site is ready it would be good to at least split it up into multiple pages or markdown to make it a bit more digestible.

**Comment by leighhalliday** [comment]
Hey Mark! We (at Wrapbook) are looking to use Pages for our Design System documentation. Because we'd like to have some info (let's say about an Avatar) for Design, other info for Web developers who are going to be using the Avatar component, and maybe more info for how you'd do it in our React Native app, it would be great to split that up into tabs... similar to how Shopify's Polaris or Material does it below:

![Screen Shot 2022-06-02 at 2 00 23 PM](https://user-images.githubusercontent.com/603921/171868170-6237e8dd-f871-4f25-9b19-a560c890c581.png)

![Screen Shot 2022-06-02 at 1 58 16 PM](https://user-images.githubusercontent.com/603921/171868201-f3b9ca03-3c34-4656-8196-b654bd6b35bd.png)

I wonder if there is something around file naming where we could make it work:
- avatar[design].html.erb
- avatar[web].html.erb
- avatar[mobile].html.erb

And when it is detecting/loading pages, it knows they are the same page, but splits them up into 3 tabs? Just brainstorming! We are happy to (and probably will) explore the idea, but wanted to see if you had thoughts about this!

**Comment by allmarkedup** [comment]
Hey @leighhalliday - great to hear that you are planning on using Lookbook :-)

This issue was actually about splitting up the Lookbook documentation from one giant README rather than about the Pages feature, but what you are describing makes total sense - I'd not really thought about splitting up pages like that but I can see that could be very useful for scenarios like the one you described.

I'm not sure if it makes sense to go down a very specific implementation route for that sort of thing just yet, but I have got plans for allowing more customisation of the page controller/layout/templates which I could definitely see extending to supporting things like tabs and pulling in content in a more flexible way. So I'm confident that something like this can be supported in the not-too-distant future. The pages are really just ERB files at the end of the day so there is a lot of scope for extending their functionality.

Thanks for the suggestion and I'll try to let you know when I've implemented anything that might be of use to you 👍
